pep-0517.txt

```diff
@@ -923,6 +1002,15 @@ above, there are a few other differences in this proposal:
   but these aren't normative anyway.
 
 
+==================
+ Revision History
+==================
+
+- v1.0 - Initial Release
+- v2.0 - Added the ``python-paths`` key, restrictions on requirement
```

Comment:
\<bikeshed>

Should this be 1.1 since it's backwards compatible change here? If not, is there any reason to keep a 2-part version number? Should we include dates here?

\</bikeshed>

I suggest that this be reworded something similar to PEP 440's changes summary instead -- https://www.python.org/dev/peps/pep-0440/#summary-of-changes-to-pep-440

Reply:
I made that whole bit up, just because it's a relatively substantial change to the PEP, so I thought a note was worth adding.

Thanks for the link to PEP 440's approach, I'll follow that - consistency of approach is probably the most important thing here.